dev/src-expose/snapshot.go

```diff
@@ -13,10 +13,35 @@ import (
 	"github.com/sourcegraph/sourcegraph/lib/errors"
 )
 
-func snapshot(logger *log.Logger, src, dst string) error {
-	name := filepath.Base(src)
+/*
+ * Create a gitignore file in ${GIT_DIR}/info/exclude
+ * if there are ignores defined in the configuration file.
+ * Returns `error` if the file could not be created or written to.
+ */
+func processIgnores(ignores []string, dst string) error {
+	if ignores != nil {
+		gitignore_path := filepath.Join(dst, "info", "exclude")
```

Comment:
Go style is camelCase :)

```suggestion
		gitignorePath := filepath.Join(dst, "info", "exclude")
```